### ISSUE TITLE
360° Youtube videos show black screen when using vp9

### ISSUE THREAD

**Issue by asemarafa** [xtknight/vdpau-va-driver-vp9#8]
Hello 

As per the title description, if you try to open any 360° video in youtube you will only have a black screen.

If I switch to h264 the video works normally, so can you have a look at this?

Error is here http://ix.io/2hYh

**Comment by xtknight** [comment]
Thanks for the issue report and debug logs. I'll see if there's anything I can do, although unsure how 360 degree videos work exactly.

**Comment by taktiker12** [comment]
go to "chrome://flags" and search for  #disable-accelerated-video-decode ,then disable it .
it works for me

**Comment by beew** [comment]
> go to "chrome://flags" and search for #disable-accelerated-video-decode ,then disable it .
> it works for me


Yes of course but then the whole point of this project is to enable accelerate video decode.
